ios/Divoc/Imported/Networking/Reporting/ParametersManager.swift

```diff
@@ -119,12 +121,32 @@ private class ParametersFetchOperation: Operation {
 
           DP3TTracing.parameters = dp3tParameters
         }
+
         if let appversion = dictionary["versions"] as? Dictionary<String,Dictionary<String,String>> {
           let iosversion = appversion["ios"] ?? [:]
           let version = iosversion["name"] ?? "0"
           let build = iosversion["build"] ?? "0"
           AppVersionManager.shared.checkAppVersion(version: version,buildno: build)
         }
+
+        if let infobox = dictionary["infoBox"] as? Dictionary<String, Dictionary<String, Any>> {
+          let info = infobox[.languageKey] ?? nil
```

Comment:
Again, here you could wrap this in the same if let and just drop the `!` below when accessing info.
```
if
    let infobox = dictionary["infoBox"] as? Dictionary<String, Dictionary<String, Any>>,
    let info = infobox[.languageKey] {
    ......
}
```

Reply:
Right, we will change before merging.